packages/website/src/components/editor/LoadedEditor.tsx

```diff
@@ -101,19 +113,19 @@ export const LoadedEditor: React.FC<LoadedEditorProps> = ({
     );
     webLinter.updateCompilerOptions(config);
     sandboxInstance.setCompilerSettings(config);
-  }, [jsx, tsconfig]);
+  }, [jsx, sandboxInstance, tsconfig, webLinter]);
 
   useEffect(() => {
     webLinter.updateRules(parseESLintRC(eslintrc).rules);
-  }, [eslintrc]);
+  }, [eslintrc, webLinter]);
 
   useEffect(() => {
     sandboxInstance.editor.setModel(tabs[activeTab]);
     updateMarkers();
-  }, [activeTab]);
+  }, [activeTab, sandboxInstance.editor, tabs, updateMarkers]);
 
-  useEffect(
-    debounce(() => {
+  useEffect(() => {
+    const lintEditor = debounce(() => {
```

Comment:
I've updated this hook because otherwise ESLint cannot parse correctly its dependencies and throws a warning. The functionality remains the same:

<img width="733" alt="Screen Shot 2022-09-27 at 9 03 04 PM" src="https://user-images.githubusercontent.com/36057168/192664498-3fda23ac-578f-44a2-89e2-0a7e818eda4a.png">